driver/windows/pwnto-driver/example/ExampleComPort/ExampleComPort.vcxproj

```diff
@@ -0,0 +1,144 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
could you reuse my makefile please? Sounds better than  windows proprietary xml file for compilation.

Reply:
my bad. I just realized it was the example driver.